README.md

```diff
@@ -221,17 +267,33 @@ val df = spark.read.format("com.cognite.spark.datasource")
   .option("type", "events")
   .load()
 
+// Insert the events in your own project using .save()
+import org.apache.spark.sql.functions._
+df.withColumn("source", lit("publicdata"))
+  .write.format("com.cognite.spark.datasource")
+  .option("apiKey", "myApiKey")
+  .option("onconflict", "abort")
+  .save()
+
 // Get a reference to the events in your project
 val myProjectDf = spark.read.format("com.cognite.spark.datasource")
   .option("apiKey", "myApiKey")
   .option("type", "events")
   .load()
 myProjectDf.createTempView("events")
 
-// Copy the Valhall events to your project
-df.filter($"subtype" === "Valhall")
-  .write
-  .insertInto("events")
+// Update the description of all events from Open Industrial Data
+spark.sql("""
+ |select 'Manually copied data from publicdata' as description,
```

Comment:
GitHub highlights this strangely, might be a bug on their side. Just make sure it renders ok before merging.

Reply:
Yeah, noticed this as well. It's their syntax highlighting that's off, it seems. I guess ```scala doesn't like triple quoted strings or something.